manual/_static/toc.js

```
@@ -0,0 +1,23 @@
+  if (!toc) return;
+
+  // "Unindent" the table of contents.
+  const tocItems = toc.querySelector("ul");
+  if (tocItems) {
+    const tocInnerItems = tocItems.querySelector("li ul");
+    tocInnerItems && toc.replaceChild(tocInnerItems, tocItems);
+  }
+
+  // Move the table of contents after the title.
+  const titleHeading = document.querySelector(".document h1");
+  titleHeading && titleHeading.insertAdjacentElement("afterend", toc);
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)